src/wallet/test/init_tests.cpp

```diff
@@ -7,6 +7,10 @@
 #include <test/setup_common.h>
 #include <wallet/test/init_test_fixture.h>
 
+extern void noui_connect();
```

Comment:
Why are these `extern void`s here? Doesn't including the header that defines them (`noui.h`) suffice?

Reply:
Given the noui_connect() was there when I started, I followed the same convention. Have changed it now, removed the `extern void`s and included `noui.h` in stead.